docs/dom-testing-library/api-helpers.md

```diff
@@ -148,6 +148,30 @@ cy.get('form').within(() => {
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+## `getRoles`
+
+This function allows iteration over the implicit aria roles represented in a
```

Comment:
```suggestion
This function allows iteration over the implicit ARIA roles represented in a
```

(and same suggestion in other places where ARIA is lowercased!)

Reply:
Done [3390faa](https://github.com/testing-library/testing-library-docs/pull/155/commits/3390faabbafc4bc3c232c40b41f7c8a931099665)